                                                                                                                                                                                                                                                               
Delivered-To: jwboyer@gmail.com
Received: by 10.220.45.11 with SMTP id c11cs78852vcf;
        Tue, 25 Oct 2011 11:54:49 -0700 (PDT)
Received: by 10.236.124.17 with SMTP id w17mr28007377yhh.126.1319568876490;
        Tue, 25 Oct 2011 11:54:36 -0700 (PDT)
Return-Path: <linux-kernel-owner@vger.kernel.org>
Received: from vger.kernel.org (vger.kernel.org. [209.132.180.67])
        by mx.google.com with ESMTP id d47si24837679yhn.69.2011.10.25.11.54.34;
        Tue, 25 Oct 2011 11:54:36 -0700 (PDT)
Received-SPF: pass (google.com: best guess record for domain of linux-kernel-owner@vger.kernel.org designates 209.132.180.67 as permitted sender) client-ip=209.132.180.67;
Authentication-Results: mx.google.com; spf=pass (google.com: best guess record for domain of linux-kernel-owner@vger.kernel.org designates 209.132.180.67 as permitted sender) smtp.mail=linux-kernel-owner@vger.kernel.org
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1750837Ab1JYSyG (ORCPT <rfc822;abhay.linux@gmail.com>
	+ 99 others); Tue, 25 Oct 2011 14:54:06 -0400
Received: from mx1.redhat.com ([209.132.183.28]:17177 "EHLO mx1.redhat.com"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1750755Ab1JYSyF (ORCPT <rfc822;linux-kernel@vger.kernel.org>);
	Tue, 25 Oct 2011 14:54:05 -0400
Received: from int-mx02.intmail.prod.int.phx2.redhat.com (int-mx02.intmail.prod.int.phx2.redhat.com [10.5.11.12])
	by mx1.redhat.com (8.14.4/8.14.4) with ESMTP id p9PIs3QL004267
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=OK);
	Tue, 25 Oct 2011 14:54:03 -0400
Received: from gelk.kernelslacker.org (ovpn-113-62.phx2.redhat.com [10.3.113.62])
	by int-mx02.intmail.prod.int.phx2.redhat.com (8.13.8/8.13.8) with ESMTP id p9PIs2CE024426
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NO);
	Tue, 25 Oct 2011 14:54:03 -0400
Received: from gelk.kernelslacker.org (gelk [127.0.0.1])
	by gelk.kernelslacker.org (8.14.5/8.14.4) with ESMTP id p9PIs0Ia021463;
	Tue, 25 Oct 2011 14:54:01 -0400
Received: (from davej@localhost)
	by gelk.kernelslacker.org (8.14.5/8.14.5/Submit) id p9PIs0wZ021462;
	Tue, 25 Oct 2011 14:54:00 -0400
X-Authentication-Warning: gelk.kernelslacker.org: davej set sender to davej@redhat.com using -f
Date:	Tue, 25 Oct 2011 14:54:00 -0400
From:	Dave Jones <davej@redhat.com>
To:	Mark Brown <broonie@opensource.wolfsonmicro.com>
Cc:	Linux Kernel <linux-kernel@vger.kernel.org>
Subject: wm8400-core: select REGMAP_I2C in kconfig
Message-ID: <20111025185359.GA21019@redhat.com>
Mail-Followup-To: Dave Jones <davej@redhat.com>,
	Mark Brown <broonie@opensource.wolfsonmicro.com>,
	Linux Kernel <linux-kernel@vger.kernel.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
User-Agent: Mutt/1.5.21 (2010-09-15)
X-Scanned-By: MIMEDefang 2.67 on 10.5.11.12
Sender:	linux-kernel-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-kernel.vger.kernel.org>
X-Mailing-List:	linux-kernel@vger.kernel.org

wm8400-core.c is using the regmap I2C API, so needs to select it.

ERROR: "regmap_write" [drivers/mfd/wm8400-core.ko] undefined!
ERROR: "regmap_raw_read" [drivers/mfd/wm8400-core.ko] undefined!
ERROR: "regmap_read" [drivers/mfd/wm8400-core.ko] undefined!
ERROR: "regmap_init_i2c" [drivers/mfd/wm8400-core.ko] undefined!
ERROR: "regmap_bulk_read" [drivers/mfd/wm8400-core.ko] undefined!
ERROR: "regmap_exit" [drivers/mfd/wm8400-core.ko] undefined!

Signed-off-by: Dave Jones <davej@redhat.com>

diff --git a/drivers/mfd/Kconfig b/drivers/mfd/Kconfig
index a67adcb..67a6126 100644
--- a/drivers/mfd/Kconfig
+++ b/drivers/mfd/Kconfig
@@ -389,6 +389,7 @@ config MFD_MAX8998
 config MFD_WM8400
 	tristate "Support Wolfson Microelectronics WM8400"
 	select MFD_CORE
+	select REGMAP_I2C
 	depends on I2C
 	help
 	  Support for the Wolfson Microelecronics WM8400 PMIC and audio
--
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/
